recipes/pyamg/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```

Comment:
Could we please update this to match [this]( https://github.com/conda-forge/pytest-benchmark-feedstock/blob/master/recipe/bld.bat#L1 )?